.github/workflows/ci.yml

```diff
@@ -647,3 +647,31 @@ jobs:
           ./scripts/typos --format json | tee /tmp/typos.json | jq -rs '.[] | "::error file=\(.path),line=\(.line_num),col=\(.byte_offset)::\(.typo) should be \"" + (.corrections // [] | join("\" or \"") + "\"")'
           cat /tmp/typos.json
           ! grep -q '[^[:space:]]' /tmp/typos.json
+
+  kani:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - uses: actions-rs/toolchain@v1.0.7
+        id: toolchain
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - uses: camshaft/install@v1
+        with:
+          crate: kani-verifier
+          version: 0.11.0
+          bins: kani,cargo-kani
+
+      - name: Kani setup
+        run: cargo-kani setup
+
+      - name: Kani run
+        run: |
+          cd quic/s2n-quic-core
```

Comment:
Should this be changed to `s2n-quic-transport` instead, so that [this `interval_set` harness](https://github.com/aws/s2n-quic/blob/80e5a554fea9f5ca5ed9befa8012170eaecfdd62/quic/s2n-quic-transport/src/interval_set/fuzz_target.rs#L103) is included?

Reply:
I'm going to open a separate PR to move interval set to core

Reply:
PR: https://github.com/aws/s2n-quic/pull/1526 ([Kani task](https://github.com/aws/s2n-quic/actions/runs/3223085024/jobs/5272794275.))